website/src/theme/ChecklistSummary/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {usePluginData} from '@docusaurus/useGlobalData'
+import styles from './styles.module.css'
+import { isChecked } from '../ArticleChecklist';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+function matchesTag(tags, t) {
+    return tags.filter(tag => tag.label == t).length > 0;
+}
+
+function ChecklistItem({title, item}) {
+    const checked = isChecked(title+"_"+item.title)
+    console.log("Checked: "+checked)
+    const className = checked ? styles.doneItem : styles.notDoneItem
+    return (
+        <div className={className}>
+            <p>{item.title}</p>
+        </div>
+    )
+}
+
+function completionCount(checklist, title) {
+    const items = checklist.length
+    const completedItems = checklist.filter(i => isChecked(title+"_"+i.title)).length;
```

Comment:
Best practice is to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of string concatenation:
```suggestion
    const completedItems = checklist.filter(i => isChecked(`{$title}_{$i.title}`)).length;
```

Reply:
done